client.go

```diff
@@ -220,13 +228,15 @@ func WithRuntime(name string) NewContainerOpts {
 	}
 }
 
+// WithSnapshotter sets the provided snapshotter for use by the container
```

Comment:
would it be worth repeating `when creating a container` here and for the other `NewContainerOpts`?